samples/GraphQL.DataLoader.Sample.Default/GraphQl/SalespersonGraphType.cs

```diff
@@ -0,0 +1,25 @@
+using GraphQL.DataLoader.Di.Sample.Types;
+using GraphQL.Types;
+using Microsoft.EntityFrameworkCore;
+
+namespace GraphQL.DataLoader.Di.Sample.GraphQl;
+
+public sealed class SalespersonGraphType : ObjectGraphType<Salesperson>
+{
+    public SalespersonGraphType(IDataLoaderContextAccessor accessor, DealershipDbContext dbContext)
+    {
+        Field(x => x.Id, type: typeof(IntGraphType)).Description("Id of the salesman");
+        Field(x => x.Name, type: typeof(StringGraphType)).Description("Name of the salesman");
+
+        Field<ListGraphType<CarsGraphType>, IEnumerable<Car>>("assignedCars").Description("Assigned cars")
```

Comment:
```suggestion
        Field<IEnumerable<Car>>("assignedCars", false).Description("Assigned cars")
```
i think this is correct...